.github/workflows/cargo-vendor.yml

```diff
@@ -18,6 +18,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Checkout submodules
+        run: |
+          git config --global --add safe.directory '*'
+          git submodule update --init --depth=1 dist-assets/binaries wireguard-go-rs/libwg
```

Comment:
shouldn't this be wireguard-go-rs/libwg/**wireguard-go**?

Reply:
woops